rfcs/convergence/focus-and-iteration-towers-2.0.md

```diff
@@ -0,0 +1,95 @@
+# RFC: Your proposal name here
+
+---
+
+_contributors: @khmakoto @xugao @miroslavstastny @jurokapsiar @dzearing @levithomason @paulgildea_
+
+## Summary
+
+This RFC covers a proposal for a process to track and bring clarity to the engineering work required to build a Fluent UI Next component
+
+## Background
+
+We have been working for a bit on convergence and leaning the towers between libraries formerly known as Stardust and Fabric. We've made good progress on defining the [component model](https://github.com/microsoft/fluentui/tree/master/packages/react-compose/src/next) to use, and are now making progress on styling and theming components built upon it.
```

Comment:
The naming of this package might probably have been not great, but we broke out the `next` of `react-compose` here and the current link is now dead

```suggestion
We have been working for a bit on convergence and leaning the towers between libraries formerly known as Stardust and Fabric. We've made good progress on defining the [component model](https://github.com/microsoft/fluentui/blob/master/packages/react-utils) to use, and are now making progress on styling and theming components built upon it.
```